pyproject.toml

```diff
@@ -56,6 +56,8 @@ tox = "^3.24"
 click = "^8.1"
 Mako = "^1.2"
 deprecation = "^2.1"
+ni-python-styleguide = "^0.3"
+black = "^23.1.0"
```

Comment:
I don't think we need to explicitly specify a dependency on `black`. `ni-python-styleguide` depends on `black`, so it will be installed.

@mshafer-NI, please correct me if I'm wrong.